pubspec.yaml

```
@@ -5,7 +5,7 @@ repository: https://github.com/woltapp/wolt_modal_sheet
 
 environment:
   sdk: '>=3.0.6 <4.0.0'
-  flutter: 3.10.6
+  flutter: '>=3.13.0'
```

```suggestion
  # Provide explicit version for flutter-action to automatically pull this version.
  flutter: 3.13.0 
```
Might be worth documentating for future releases